daal4py/sklearn/_utils.py

```diff
@@ -151,3 +152,113 @@ def get_number_of_types(dataframe):
         return len(set(dtypes))
     except TypeError:
         return 1
+
+
+def _create_global_queue_from_sycl_context():
+    import sys
+    d4p_target = None
+    if 'daal4py.oneapi' in sys.modules:
+        from daal4py.oneapi import _get_device_name_sycl_ctxt
+        d4p_target = _get_device_name_sycl_ctxt()
+
+    if d4p_target is not None:
```

Comment:
Will it work correctly if there are two identical devices, for example 2 GPUs? Because you create a new queue knowing only the name of the device